src/__tests__/CompareResults/RevisionRow.test.tsx

```diff
@@ -1,9 +1,7 @@
-import { screen } from '@testing-library/react';
```

Comment:
note: I also took the opportunity to bring together all testing-library-related things, and import them all from `test-utils`.